.github/CODEOWNERS

```diff
@@ -0,0 +1,8 @@
+# Team Lead
+* @rosle
+
+# Team Members
+* @andyduong1920 @bterone @byhbt @carryall @hanam1ni @hoangmirs @junan @Lahphim @longnd @malparty @tyrro
+
+# Engineering Leads
+CODEOWNERS @nimblehq/engineering-leads
```

Comment:
I already went through the [Github Doc](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners) regarding code owner but still don't get the syntax here.
basically, all Github users + nimble EL defined in the file are the code owners. But why do we need the `CODEOWNERS` before the EL team, will it work if we remove that keyword?

Reply:
CODEOWNERS is the file name so this to ensure that any changes to that file will require the review of ELs 💪

Reply:
it's the syntax of `codeowner` file
```
`*` will request the review of the changes on all files
`CODEOWNERS` will request the review from the ELs on the CODEOWNERS file itself.
```

thanks Olivier for clarifying.